packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewer.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withTheme } from 'styled-components'
+import { Bar } from '@vx/shape'
+import { Group } from '@vx/group'
+import { AxisBottom } from '@vx/axis'
+import { scaleBand, scaleLinear } from '@vx/scale'
+import Chart from '../SVGComponents/Chart'
+import Background from '../SVGComponents/Background'
+
+const BarChartViewer = React.forwardRef(function BarChartViewer({ backgroundFill, data, height, theme, width }, ref) {
+  const yMax = height * 0.95
```

Comment:
I'd suggest keeping constant values like this in a config file for the component. It's not too much of an issue here, but I can imagine complex charts getting a bit hairy with where these are defined